python: can't open file 'make_many_databases.py': [Errno 2] No such file or directory
